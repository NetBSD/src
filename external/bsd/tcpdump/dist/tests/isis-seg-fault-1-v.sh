#!/bin/sh

# This "verbose" ISIS protocol test involves a float calculation that
# may produce a slightly different result if the compiler is not GCC.
# Test only with GCC (similar to GitHub issue #333).

exitcode=0
test_name=isis-seg-fault-1-v

srcdir=${1-..}
: echo $0 using ${srcdir}

testdir=${srcdir}/tests
passedfile=tests/.passed
failedfile=tests/.failed
passed=`cat ${passedfile}`
failed=`cat ${failedfile}`

if [ ! -f Makefile ]
then
	printf '    %-35s: TEST SKIPPED (no Makefile)\n' $test_name
elif grep '^CC = .*gcc' Makefile >/dev/null
then
	if ${testdir}/TESTonce $test_name ${testdir}/isis-seg-fault-1.pcap ${testdir}/isis-seg-fault-1-v.out '-v'
	then
		passed=`expr $passed + 1`
		echo $passed >${passedfile}
	else
		failed=`expr $failed + 1`
		echo $failed >${failedfile}
		exitcode=1
	fi
else
	printf '    %-35s: TEST SKIPPED (compiler is not GCC)\n' $test_name
fi

exit $exitcode
