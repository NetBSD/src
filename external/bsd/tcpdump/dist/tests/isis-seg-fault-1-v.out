IS-IS, length 1497
	L2 Lan IIH, hlen: 27, v: 1, pdu-v: 1, sys-id-len: 6 (0), max-area: 3 (0)
	  source-id: 4444.0444.4444,  holding time: 10s, Flags: [Level 2 only]
	  lan-id:    6344.4444.4444.01, Priority: 64, PDU length: 1497
	    Protocols supported TLV #129, length: 1
	      NLPID(s): IPv4 (0xcc)
	    Area address(es) TLV #1, length: 4
	      Area address (length: 3): 49.0014
	    unknown TLV #80, length: 4
		0x0000:  0a00 0002
	    Restart Signaling TLV #211, length: 3
	      Flags [none], Remaining holding time 0s
	    IS Neighbor(s) TLV #6, length: 6
	      SNPA: c202.2998.0051
	    Padding TLV #8, length: 255
	    IS Alias ID TLV #24, length: 255
	      IS Neighbor: f500.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, no sub-TLVs present
	      IS Neighbor: 0000.0000.0000.00, sub-TLVs present (37)
	        unknown subTLV #0, length: 0
	        unknown subTLV #0, length: 0
	        unknown subTLV #0, length: 0
	        unknown subTLV #0, length: 0
	        Bandwidth Constraints subTLV #22, length: 0
	        Bandwidth Constraints Model ID: Russian dolls (0)
	          Bandwidth constraint CT0: 0.000 Mbps
	          Bandwidth constraint CT1: 0.000 Mbps
	          Bandwidth constraint CT2: 0.000 Mbps
	          Bandwidth constraint CT3: 0.000 Mbps
	          Bandwidth constraint CT4: 0.000 Mbps
	          Bandwidth constraint CT5: 0.000 Mbps
	          Bandwidth constraint CT6: -0.000 Mbps
	          Bandwidth constraint CT7: 0.000 Mbps
	          Bandwidth constraint CT8: 0.000 Mbps
	          Bandwidth constraint CT9: 0.000 Mbps
	          Bandwidth constraint CT10: 0.000 Mbps
	          Bandwidth constraint CT11: 0.000 Mbps
	          Bandwidth constraint CT12: 0.000 Mbps
	          Bandwidth constraint CT13: 0.000 Mbps
	          Bandwidth constraint CT14: -1267650597027719304708096.000 Mbps
	          Bandwidth constraint CT15: 0.000 Mbps
	          Bandwidth constraint CT16: 0.000 Mbps
	          Bandwidth constraint CT17: 0.000 Mbps
	          Bandwidth constraint CT18: 0.000 Mbps
	          Bandwidth constraint CT19: 0.000 Mbps
	          Bandwidth constraint CT20: 0.000 Mbps
	          Bandwidth constraint CT21: 0.000 Mbps
	          Bandwidth constraint CT22: 0.000 Mbps
	          Bandwidth constraint CT23: 0.000 Mbps
	          Bandwidth constraint CT24: 0.000 Mbps
	          Bandwidth constraint CT25: 0.000 Mbps
	          Bandwidth constraint CT26: 0.000 Mbps
	          Bandwidth constraint CT27: 0.000 Mbps
	          Bandwidth constraint CT28: 0.000 Mbps
	          Bandwidth constraint CT29: 0.000 Mbps
	          Bandwidth constraint CT30: 0.000 Mbps
	          Bandwidth constraint CT31: 0.000 Mbps
	          Bandwidth constraint CT32: 0.000 Mbps
	          Bandwidth constraint CT33: 0.000 Mbps
	          Bandwidth constraint CT34: 0.000 Mbps
	          Bandwidth constraint CT35: 0.000 Mbps
	          Bandwidth constraint CT36: 0.000 Mbps
	          Bandwidth constraint CT37: 0.000 Mbps
	          Bandwidth constraint CT38: 16.777 Mbps
	          Bandwidth constraint CT39: 0.000 Mbps
	          Bandwidth constraint CT40: 0.000 Mbps
	          Bandwidth constraint CT41: 0.000 Mbps
	          Bandwidth constraint CT42: 0.000 Mbps
	          Bandwidth constraint CT43: 0.000 Mbps
	          Bandwidth constraint CT44: 0.000 Mbps
	          Bandwidth constraint CT45: 0.000 Mbps
	          Bandwidth constraint CT46: 0.000 Mbps
	          Bandwidth constraint CT47: 0.000 Mbps
	          Bandwidth constraint CT48: 0.000 Mbps
	          Bandwidth constraint CT49: 0.000 Mbps
	          Bandwidth constraint CT50: 0.000 Mbps
	          Bandwidth constraint CT51: 0.000 Mbps
	          Bandwidth constraint CT52: 0.000 Mbps
	          Bandwidth constraint CT53: 0.000 Mbps
	          Bandwidth constraint CT54: 0.000 Mbps
	          Bandwidth constraint CT55: 0.000 Mbps
	          Bandwidth constraint CT56: 0.000 Mbps
	          Bandwidth constraint CT57: 0.000 Mbps
	          Bandwidth constraint CT58: 0.000 Mbps
	          Bandwidth constraint CT59: 0.000 Mbps
	          Bandwidth constraint CT60: 0.000 Mbps
	          Bandwidth constraint CT61: 0.000 Mbps
	          Bandwidth constraint CT62: 0.000 Mbps
	          Bandwidth constraint CT63: 0.000 Mbps
	          Bandwidth constraint CT64: 0.000 Mbps
	          Bandwidth constraint CT65: 0.000 Mbps
	          Bandwidth constraint CT66: 0.000 Mbps
	          Bandwidth constraint CT67: 0.000 Mbps
	          Bandwidth constraint CT68: 0.000 Mbps
	          Bandwidth constraint CT69: 0.000 Mbps
	          Bandwidth constraint CT70: 0.000 Mbps
	          Bandwidth constraint CT71: 0.000 Mbps
	          Bandwidth constraint CT72: 0.000 Mbps
	          Bandwidth constraint CT73: 0.000 Mbps
	          Bandwidth constraint CT74: 0.000 Mbps
	          Bandwidth constraint CT75: 0.000 Mbps
	          Bandwidth constraint CT76: 0.000 Mbps
	          Bandwidth constraint CT77: 0.000 Mbps
	          Bandwidth constraint CT78: 0.000 Mbps
	          Bandwidth constraint CT79: 0.000 Mbps
	          Bandwidth constraint CT80: 0.000 Mbps
	          Bandwidth constraint CT81: 0.000 Mbps
	          Bandwidth constraint CT82: 0.000 Mbps
	          Bandwidth constraint CT83: 0.000 Mbps
	          Bandwidth constraint CT84: 0.000 Mbps
	          Bandwidth constraint CT85: 0.000 Mbps
	          Bandwidth constraint CT86: 0.000 Mbps
	          Bandwidth constraint CT87: 0.000 Mbps
	          Bandwidth constraint CT88: 0.000 Mbps
	          Bandwidth constraint CT89: 0.000 Mbps
	          Bandwidth constraint CT90: 0.000 Mbps
	          Bandwidth constraint CT91: 0.000 Mbps
	          Bandwidth constraint CT92: 0.000 Mbps
	          Bandwidth constraint CT93: 0.000 Mbps
	          Bandwidth constraint CT94: 0.000 Mbps
	          Bandwidth constraint CT95: 0.000 Mbps
	          Bandwidth constraint CT96: 0.000 Mbps
	          Bandwidth constraint CT97: 0.000 Mbps
	          Bandwidth constraint CT98: 0.000 Mbps
	          Bandwidth constraint CT99: 0.000 Mbps
	          Bandwidth constraint CT100: 0.000 Mbps
	          Bandwidth constraint CT101: 0.000 Mbps
	          Bandwidth constraint CT102: 0.000 Mbps
	          Bandwidth constraint CT103: 0.000 Mbps
	          Bandwidth constraint CT104: 0.000 Mbps
	          Bandwidth constraint CT105: 0.000 Mbps
	          Bandwidth constraint CT106: 0.000 Mbps
	          Bandwidth constraint CT107: 0.000 Mbps
	          Bandwidth constraint CT108: 0.000 Mbps
	          Bandwidth constraint CT109: 0.000 Mbps
	          Bandwidth constraint CT110: 334903128525925002608918921216.000 Mbps
	          Bandwidth constraint CT111: 0.000 Mbps
	          Bandwidth constraint CT112: 0.000 Mbps
	          Bandwidth constraint CT113: 0.000 Mbps
	          Bandwidth constraint CT114: 0.000 Mbps
	          Bandwidth constraint CT115: 0.000 Mbps
	          Bandwidth constraint CT116: 0.000 Mbps
	          Bandwidth constraint CT117: 0.000 Mbps
	          Bandwidth constraint CT118: 0.000 Mbps
	          Bandwidth constraint CT119: 0.000 Mbps
	          Bandwidth constraint CT120: 0.000 Mbps
	          Bandwidth constraint CT121: 0.000 Mbps
	          Bandwidth constraint CT122: 0.000 Mbps
	          Bandwidth constraint CT123: 0.000 Mbps
	          Bandwidth constraint CT124: 0.000 Mbps
	          Bandwidth constraint CT125: 0.000 Mbps
	          Bandwidth constraint CT126: 0.000 Mbps
	          Bandwidth constraint CT127: 0.000 Mbps
	          Bandwidth constraint CT128: 0.000 Mbps
	          Bandwidth constraint CT129: 0.000 Mbps
	          Bandwidth constraint CT130: 0.000 Mbps
	          Bandwidth constraint CT131: 0.000 Mbps
	          Bandwidth constraint CT132: 0.000 Mbps
	          Bandwidth constraint CT133: 0.000 Mbps
	          Bandwidth constraint CT134: 0.000 Mbps
	          Bandwidth constraint CT135: 0.000 Mbps
	          Bandwidth constraint CT136: 0.000 Mbps
	          Bandwidth constraint CT137: -4.194 Mbps
	          Bandwidth constraint CT138: 0.000 Mbps
	          Bandwidth constraint CT139: 0.000 Mbps
	          Bandwidth constraint CT140: 0.000 Mbps
	          Bandwidth constraint CT141: 0.000 Mbps
	          Bandwidth constraint CT142: 0.000 Mbps
	          Bandwidth constraint CT143: 0.000 Mbps
	          Bandwidth constraint CT144: 0.000 Mbps
	          Bandwidth constraint CT145: 0.000 Mbps
	          Bandwidth constraint CT146: 0.000 Mbps
	          Bandwidth constraint CT147: 0.000 Mbps
	          Bandwidth constraint CT148: 0.000 Mbps
	          Bandwidth constraint CT149: 0.000 Mbps
	          Bandwidth constraint CT150: 0.000 Mbps
	          Bandwidth constraint CT151: 0.000 Mbps
	          Bandwidth constraint CT152: 0.000 Mbps
	          Bandwidth constraint CT153: 0.000 Mbps
	          Bandwidth constraint CT154: 16.777 Mbps
	          Bandwidth constraint CT155: 0.000 Mbps
	          Bandwidth constraint CT156: 0.000 Mbps
	          Bandwidth constraint CT157: 0.000 Mbps
	          Bandwidth constraint CT158: 0.000 Mbps
	          Bandwidth constraint CT159: 0.000 Mbps
	          Bandwidth constraint CT160: 0.000 Mbps
	          Bandwidth constraint CT161: 0.000 Mbps
	          Bandwidth constraint CT162: 0.000 Mbps
	          Bandwidth constraint CT163: 0.000 Mbps
	          Bandwidth constraint CT164: 0.000 Mbps
	          Bandwidth constraint CT165: 0.000 Mbps
	          Bandwidth constraint CT166: 0.000 Mbps
	          Bandwidth constraint CT167: 0.000 Mbps
	          Bandwidth constraint CT168: 0.000 Mbps
	          Bandwidth constraint CT169: 0.000 Mbps
	          Bandwidth constraint CT170: 0.000 Mbps
	          Bandwidth constraint CT171: 0.000 Mbps
	          Bandwidth constraint CT172: 0.000 Mbps
	          Bandwidth constraint CT173: 0.000 Mbps
	          Bandwidth constraint CT174: 0.000 Mbps
	          Bandwidth constraint CT175: 0.000 Mbps
	          Bandwidth constraint CT176: 0.000 Mbps
	          Bandwidth constraint CT177: 0.000 Mbps
	          Bandwidth constraint CT178: 0.000 Mbps
	          Bandwidth constraint CT179: 1099511.625 Mbps
	          Bandwidth constraint CT180: 0.000 Mbps
	          Bandwidth constraint CT181: 0.000 Mbps
	          Bandwidth constraint CT182: 0.000 Mbps
	          Bandwidth constraint CT183: 0.000 Mbps
	          Bandwidth constraint CT184: 0.000 Mbps
	          Bandwidth constraint CT185: 0.000 Mbps
	          Bandwidth constraint CT186: 0.000 Mbps
	          Bandwidth constraint CT187: 0.000 Mbps
	          Bandwidth constraint CT188: 0.000 Mbps
	          Bandwidth constraint CT189: 0.000 Mbps
	          Bandwidth constraint CT190: 0.000 Mbps
	          Bandwidth constraint CT191: 0.000 Mbps
	          Bandwidth constraint CT192: 0.000 Mbps
	          Bandwidth constraint CT193: 0.000 Mbps
	          Bandwidth constraint CT194: 0.000 Mbps
	          Bandwidth constraint CT195: 0.000 Mbps
	          Bandwidth constraint CT196: 0.000 Mbps
	          Bandwidth constraint CT197: 0.000 Mbps
	          Bandwidth constraint CT198: 0.000 Mbps
	          Bandwidth constraint CT199: 0.000 Mbps
	          Bandwidth constraint CT200: 0.000 Mbps
	          Bandwidth constraint CT201: 0.000 Mbps
	          Bandwidth constraint CT202: 0.000 Mbps
	          Bandwidth constraint CT203: 0.000 Mbps
	          Bandwidth constraint CT204: 0.000 Mbps
	          Bandwidth constraint CT205: 0.000 Mbps
	          Bandwidth constraint CT206: 0.000 Mbps
	          Bandwidth constraint CT207: 0.000 Mbps
	          Bandwidth constraint CT208: 0.000 Mbps
	          Bandwidth constraint CT209: 0.000 Mbps
	          Bandwidth constraint CT210: 0.000 Mbps
	          Bandwidth constraint CT211: 0.000 Mbps
	          Bandwidth constraint CT212: 0.000 Mbps
	          Bandwidth constraint CT213: 0.000 Mbps
	          Bandwidth constraint CT214: 0.000 Mbps
	          Bandwidth constraint CT215: 0.000 Mbps
	          Bandwidth constraint CT216: 0.000 Mbps
	          Bandwidth constraint CT217: 0.000 Mbps
	          Bandwidth constraint CT218: 0.000 Mbps
	          Bandwidth constraint CT219: 0.000 Mbps
	          Bandwidth constraint CT220: 0.000 Mbps
	          Bandwidth constraint CT221: 0.000 Mbps
	          Bandwidth constraint CT222: 0.000 Mbps
	          Bandwidth constraint CT223: 0.000 Mbps
	          Bandwidth constraint CT224: 0.000 Mbps
	          Bandwidth constraint CT225: 0.000 Mbps
	          Bandwidth constraint CT226: 0.000 Mbps
	          Bandwidth constraint CT227: 0.000 Mbps
	          Bandwidth constraint CT228: 0.000 Mbps
	          Bandwidth constraint CT229: 0.000 Mbps
	          Bandwidth constraint CT230: -0.000 Mbps
	          Bandwidth constraint CT231: 0.000 Mbps
	          Bandwidth constraint CT232: 0.000 Mbps
	          Bandwidth constraint CT233: 0.000 Mbps
	          Bandwidth constraint CT234: 0.000 Mbps
	          Bandwidth constraint CT235: 0.000 Mbps
	          Bandwidth constraint CT236: 0.000 Mbps
	          Bandwidth constraint CT237: 0.000 Mbps
	          Bandwidth constraint CT238: 0.000 Mbps
	          Bandwidth constraint CT239: 0.000 Mbps
	          Bandwidth constraint CT240: 0.000 Mbps
	          Bandwidth constraint CT241: 0.000 Mbps
	          Bandwidth constraint CT242: 0.000 Mbps
	          Bandwidth constraint CT243: 0.000 Mbps
	          Bandwidth constraint CT244: 0.000 Mbps
	          Bandwidth constraint CT245: 0.000 Mbps
	          Bandwidth constraint CT246: 0.000 Mbps
	          Bandwidth constraint CT247: 0.000 Mbps
	          Bandwidth constraint CT248: 0.000 Mbps
	          Bandwidth constraint CT249: 0.000 Mbps
	          Bandwidth constraint CT250: 0.000 Mbps
	          Bandwidth constraint CT251: 0.000 Mbps
	          Bandwidth constraint CT252: 0.000 Mbps
	          Bandwidth constraint CT253: 0.000 Mbps
	          Bandwidth constraint CT254: 0.000 Mbps
	          Bandwidth constraint CT255: 0.000 Mbps
	          Bandwidth constraint CT256: 0.000 Mbps
	          Bandwidth constraint CT257: 0.000 Mbps
	          Bandwidth constraint CT258: 0.000 Mbps
	          Bandwidth constraint CT259: 0.000 Mbps
	          Bandwidth constraint CT260: 0.000 Mbps
	          Bandwidth constraint CT261: 0.000 Mbps
	          Bandwidth constraint CT262: 0.000 Mbps
	          Bandwidth constraint CT263: 0.000 Mbps
	          Bandwidth constraint CT264: 0.000 Mbps
	          Bandwidth constraint CT265: 0.000 Mbps
	          Bandwidth constraint CT266: 0.000 Mbps
	          Bandwidth constraint CT267: 0.000 Mbps
	          Bandwidth constraint CT268: 0.000 Mbps
		 [|isis]
